classes/tasks/OpenAccessNotification.inc.php

```diff
@@ -86,7 +86,7 @@ function sendNotifications ($journal, $curDate) {
 				if ($accessStatus == ISSUE_ACCESS_SUBSCRIPTION && !empty($openAccessDate) && strtotime($openAccessDate) == mktime(0,0,0,$curMonth, $curDay, $curYear)) {
 					// Notify all users who have open access notification set for this journal
 					$userSettingsDao = DAORegistry::getDAO('UserSettingsDAO');
-					$users = $userSettingsDao->getUsersBySetting('openAccessNotification', true, 'bool', $journal->getId());
+					$users = $userSettingsDao->getUsersBySetting('openAccessNotification', true, 'bool', ASSOC_TYPE_JOURNAL, $journal->getId());
```

Comment:
Check the parameter list for `UserSettingsDAO` -- not its parent class, `PKPUserSettingsDAO`.